docs/management/admin/event-filters.asciidoc

```diff
@@ -4,6 +4,8 @@
 
 Event filters allow you to filter endpoint events that you do not need or want stored in {es} -- for example, high-volume events. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.
 
+Event filters do not lower CPU usage on hosts; {elastic-endpoint} still monitors event data to detect and prevent possible threats, but the event data just isn't being written to {es}. To compare event filters with other endpoint artifacts, refer to <<endpoint-artifacts>>.
```

Comment:
```suggestion
Event filters do not lower CPU usage on hosts; {elastic-endpoint} still monitors filtered events to detect and prevent possible threats, but prevents them from being written to {es}. To compare event filters with other endpoint artifacts, refer to <<endpoint-artifacts>>.
```

Reply:
Does it write them to ES if they end up triggering an alert? 🤔

Reply:
I assume only the alerts would be written to ES. 

A modified version of your revision (and my modification here is actually based on your revision to the same phrasing in the overall artifacts table above).
```suggestion
Event filters do not lower CPU usage on hosts; {elastic-endpoint} still monitors events to detect and prevent possible threats, but without writing event data to {es}. To compare event filters with other endpoint artifacts, refer to <<endpoint-artifacts>>.
```